src_py/__init__.py

```diff
@@ -335,11 +335,11 @@ def __color_reduce(c):
 # Thanks for supporting pygame. Without support now, there won't be pygame later.
 if "PYGAME_HIDE_SUPPORT_PROMPT" not in os.environ:
     print(
-        "pygame {} (SDL {}.{}.{}, Python {}.{}.{})".format(  # pylint: disable=consider-using-f-string
+        "Pygame v{} (SDL v{}.{}.{}, Python v{}.{}.{})".format(  # pylint: disable=consider-using-f-string
```

Comment:
Lowercase p is usually preferred when referring to "pygame" as per PEP8
https://peps.python.org/pep-0008/#package-and-module-names
![image](https://user-images.githubusercontent.com/35881688/209845891-4679c41f-1bb6-4dc6-997a-e13e56a900df.png)